src/features/groups/components/InviteMemberDialog/InviteMemberDialogContent.tsx

```diff
@@ -8,32 +8,33 @@ import {
   Icon,
 } from '@/components/ui';
 import { putGenerateInvitationLinkHash } from '@/lib/api/group/client/groupApiClient.client';
+import { IGroup } from '@/types/definition';
 
 import { useEffect, useState } from 'react';
 
 const CLIENT_BASE_URL = process.env.NEXT_PUBLIC_CLIENT_BASE_URL || '';
 
-/**
- * This component displays dialog which has copy button.
- * When it is clicked, the text changes to "Copied!"
- *
- * Its state is controlled by the `isDialogOpen` prop, which is passed from the parent component.
- *
- * @param props.isDialogOpen - state to control the visibility of dialog, if it is open => true, if not => false, this state is used to switch the text in button.
- * @param groupId - a unique Id as identifier of a group
- * @returns - The JSX code for rendering the dialog component.
- */
+interface IInviteMemberDialogContentProps {
+  /**
+   * Props which is state to control the the dialog,true means open, false is closed.
+   */
+  isDialogOpen: boolean;
+  /**
+   * an identifier of a group which an invited user will belong to
+   */
+  groupId: IGroup['id'];
+}
+
 export const InviteMemberDialogContent = ({
   isDialogOpen,
   groupId,
-}: {
-  isDialogOpen: boolean;
-  groupId: string;
-}) => {
+}: IInviteMemberDialogContentProps) => {
   const [isLinkButtonClicked, setIsLinkButtonClicked] = useState(false);
 
   /**
-   * this function is onClick function when the `Button` clicked
+   * Handle the link copy button click.
+   * If the generatedLink is not valid, return.
```

Comment:
> If ..., return.
This sounds a little unnatural. Returns "what"? If you want to say "return", say "return void" or "return nothing".

But, in this situation, these expressions would sound better.
- If the generatedLink is not valid, do nothing.
- If the generatedLink is not valid, exit the function.